### DISCUSSION TITLE
Umbraco workflow - V10.4.1 - Content review overview showing duplicate nodes

### DISCUSSION THREAD

**Discussion by charlenevas21** [umbraco/Umbraco.Workflow.Issues#89] [Ideas]
Any idea why I'm seeing a duplicate of the node that has been set up for content review? (I have administration role)

![1](https://github.com/user-attachments/assets/4adea205-903a-4c02-b7df-0aaacbdcc392)
![2](https://github.com/user-attachments/assets/53df220b-da0e-496e-819b-4ec05bfe1f78)
![3](https://github.com/user-attachments/assets/0d74ab59-525c-412d-b04c-1c7c8ed2a1b4)

**Comment by ronaldbarendse** [comment]
Hi Charlene, I am looking through all discussions and noticed you didn't get any reply from us yet! 🤯 Have you encountered the same issue again and if so, also when using the latest Workflow version? We've done some improvements in handling concurrent operations, so requesting a content review for the same item at (almost) the same time shouldn't cause duplicate entries to show up.

I'll close this for now, but please follow up with more details if this still happens.
